doc/POST_INSTALL.md

```diff
@@ -0,0 +1 @@
+You now need to create the first admin user for __APP__. (FIXME: how...?)
```

Comment:
The old disclaimer did not really explain how people are supposed to "create the first admin user" x_x Doesn't prevent merging I guess, but would be nice to clarify ...